packages/expo-updates/ios/EXUpdates/Logging/UpdatesErrorCode.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+// Error codes for expo-updates logs
+
+import Foundation
+
+@objc(EXUpdatesErrorCode)
+public enum UpdatesErrorCode: Int {
+  case None = 0
+  case NoUpdatesAvailable = 1
+  case UpdateAssetsNotAvailable = 2
+  case UpdateServerUnreachable = 3
+  case UpdateHasIncorrectHash = 4
```

Comment:
it would probably be more terminologically correct to say `UpdateHasInvalidSignature` as we don't have "hashes" at the update level

Reply:
Fixed